styles.css

```diff
@@ -0,0 +1,319 @@
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-bottom: 50px;
+}
+
+.variety {
+    color: black;
+    display: flex;
+    flex-direction: row-reverse;
+    justify-content: space-between;
+    padding-bottom: 50px;
+}
+
+.worth {
+    color: black;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.worktxt {
+    display: flex;
+    flex-basis: 400px;
+    flex-direction: column;
+    margin-bottom: 25px;
+    padding: 70px;
+}
+
+.varietytxt {
+    display: flex;
+    flex-basis: 400px;
+    flex-direction: column;
+    margin-bottom: 25px;
+    padding: 70px;
+}
+
+.worthtxt {
+    display: flex;
+    flex-basis: 400px;
+    flex-direction: column;
+    margin-bottom: 25px;
+    padding: 70px;
+}
+
+.variety, .work, .worth {
+    max-width: 1457px;
+    max-height: 547px;
+}
+
+.main-images {
+    filter: drop-shadow(10px 10px 20px grey);
+}
+
+.serviceimages {
+    max-height: 100px;
+    max-width: 100px;
+}
+
+.inner-container3 h3 {
+    color: #5722CD;
+}
+
+.inner-container3 {
+    display: flex;
+    flex-wrap: wrap;
+    flex-shrink: 0;
+}
+
+.services1row1 {
+    display: flex;
+    flex-direction: row;
+    flex-basis: 350px;
+    gap: 20px;
+    padding-bottom: 30px;
+    justify-content: center;
+}
+
+.services1row2 {
+    display: flex;
+    flex-direction: row;
+    flex-basis: 350px;
+    gap: 20px;
+    padding-bottom: 30px;
+    justify-content: center;
+}
+
+.row1, .row2 {
+    border-style: groove;
+    border-color: #5722CD;
+    padding: 20px;
+    display: flex;
+    flex-basis: 300px;
+    height: 300px;
+    flex-direction: column;
+}
+
+.inner-container4 {
+    color: white;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-basis: 100%;
+    flex-shrink: 1;
+    flex-grow: 0;
+    background: rgb(73,100,232);
+    background: linear-gradient(90deg, rgba(73,100,232,1) 0%, rgba(75,73,232,1) 41%, rgba(61,34,205,1) 100%);
+    padding: 50px;
+}
+
+.form {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    flex-basis: 250px;
+    flex-shrink: 1;
+    flex-grow: 0;
+}
+
+.form input {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 1600px;
+    width: 100%;
+    height: 40px;
+    margin-bottom: 25px;
+}
+
+.messageform {
+    display: flex;
+    flex-direction: column;
+    flex-basis: 1600px;
+    width: 500px;
+    height: 200px;
+    margin-bottom: 25px;
+}
+
+.bottombutton {
+    background-color: #ED4385;
+    color: white;
+    border-style: none;
+    border-radius: 60px;
+    cursor: pointer;
+    margin-bottom: 100px;
+    padding: 25px;
+    text-align: center;
+    filter: drop-shadow(6px 5px 5px black);
+    text-transform: uppercase;
+    width: 15%;
+}
+
+
+footer {
+    font-size: 13px;
+    background-color: black;
+    color: white;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    padding: 15px;
+}
```

Comment:
Ik mis de media queries zodat het op kleine schermen (mobile) het allemaal nog op het scherm past

Reply:
Helemaal niet meer aan gedacht, heb je gelijk in.